lib/cmd/fh3/credential/download.js

```diff
@@ -0,0 +1,40 @@
+/* globals i18n */
+var fhreq = require('../../../utils/request.js');
+var fs = require('fs');
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('Download credential bundle'),
+  'examples' : [{
+    cmd : 'fhc credential download --id=<bundle-id> --output=<Path To Output File>',
```

Comment:
In order for this to work you have to pass path to existing directory with trailing slash ('/'). It's quite error prone. I would rather make this consistent with lib/cmd/fh3/appdata/export/download.js where you are expected to provide path plus filename plus extension. So instead of e.g. '/home/user/' you provide '/home/user/mybundle.zip'.